### DISCUSSION TITLE
Visual Metronome NPC ID List

### DISCUSSION THREAD

**Discussion by Puzzle-Drops** [runelite/runelite#15922] [Ideas]
Currently visual metronome is the only way I know to have a tick timer. Although the tob plugin has room tick timers only on entry. I would like a visual metronome over NPC Id's. To give an example here we have Xarpus. In order to scythe walk correctly I need to step back 2 ticks after his NPC ID changes (he's marked red). In this clip that happens at tick 3 so I need to step back on tick 1 during a 4 tick timer. During p3 Verzik I need a tick timer that is every 7 ticks instead of every 4.

Could also add an offset value so you wouldn't have text directly on top of HP percentage like in the clip shown. It could be as simple as the below:
NPCID:TickCounter:Offset
8,340:4:-24
8,372:4:+12
8,374:7:0

This plugin would take that list and as soon as NPC ID 8,340 spawns (xarpus) it would count a 4 tick timer on that NPC's location (not the player location). Another 4 tick timer for the 8,372 NPC ID (verzik p2) and then a 7 tick timer for 8,374 (verzik p3). Let me know what you think.

https://imgur.com/Xeka2RF

**Comment by Nightfirecat** [comment]
Support for hub plugins should be directed to the author of the plugin.
You can find the support link by searching for the plugin in the Plugin Hub panel and clicking the `?` button on the plugin, or by right-clicking the plugin in the plugin panel and clicking the `Support` menu option.

**Reply by Puzzle-Drops** [reply]
This isn't for the Visual Metronome plugin. It would be a new plugin itself. I'm just using that plugin to describe the behavior.
